### ISSUE TITLE
Final MIME boundary gets truncated

### ISSUE THREAD

**Issue by saeidw** [pear/Net_SMTP#34]
Hello, I'm not sure if this is a bug in Net_SMTP. We noticed that the final MIME boundary was always being truncated when communicating with an SMTP server in Net_SMTP 1.7.2 and we confirmed that the behavior is not present in 1.7.1.

In our packet capture, we observed that the final boundary is `--=_9ac6a00cd3d2ece6e38e5a24cbee6ea1-` instead of `--=_9ac6a00cd3d2ece6e38e5a24cbee6ea1--`. The second `-` is missing at the end for some reason. Downgrading to Net_SMTP 1.7.1 solved the problem and emails were being displayed correctly to the recipient.

Below you'll find a copy of the packet trace. I edited the domain names, the subject, and the `text/html` content to remove some sensitive information. There is no `text/plain` content so that section is empty.

I hope this helps reproduce the issue, and if it's not a bug in Net_SMTP then I would appreciate being pointed in the right direction.

```
220 our.domain.example.com
EHLO localhost
250-our.domain.example.com
250-PIPELINING
250-SIZE 10240000
250-ETRN
250-ENHANCEDSTATUSCODES
250-8BITMIME
250 DSN
MAIL FROM:<admin@our.domain.example.com>
250 2.1.0 Ok
RCPT TO:<saeid.wazzan@example.com>
250 2.1.5 Ok
DATA
354 End data with <CR><LF>.<CR><LF>
MIME-Version: 1.0
Content-Type: multipart/alternative;
 boundary="=_9ac6a00cd3d2ece6e38e5a24cbee6ea1"
Content-Transfer-Encoding: 8bit
From: admin@our.domain.example.com
To: saeid.wazzan@example.com
Subject: vos identifiants

--=_9ac6a00cd3d2ece6e38e5a24cbee6ea1
Content-Transfer-Encoding: 8bit
Content-Type: text/plain; charset=UTF-8


--=_9ac6a00cd3d2ece6e38e5a24cbee6ea1
Content-Transfer-Encoding: quoted-printable
Content-Type: text/html; charset=UTF-8

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns=3D"http://www.w3.org/1999/xhtml" xml:lang=3D"en">
<head>
    <meta http-equiv=3D"Content-Type" content=3D"text/html;charset=3DUTF-8"=
 />
</head>
<body style=3D"font-family: 'Arial', 'sans-serif';font-size: 10pt;">
    <p>Bonjour,</p>
</body>
</html>

--=_9ac6a00cd3d2ece6e38e5a24cbee6ea1-
.
250 2.0.0 Ok: queued as A28A244B0D
QUIT
221 2.0.0 Bye
```

**Comment by alecpl** [comment]
@jparise, I think this regression calls for a new release, please.

**Comment by jparise** [comment]
@alecpl Just released as version 1.7.3. Sorry for the delay. I was trying to get the Travis-CI build issue sorted out first, but it appears to be a problem on their end (filed as travis-ci/travis-ci#7154) as best as I can tell.
